src/commands/createVirtualMachine/windowsSteps/WindowsImageListStep.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from "vscode-azureextensionui";
```

Comment:
Please keep file-rename PRs separate from refactoring PRs. In both cases, they should be quick and easy to review _if they're split up correctly_, but very difficult to review if combined. For example, your first PR could be the refactor where you keep all the step files in "createVirtualMachine" folder and your second PR could move them into separate "windowsSteps" and "linuxSteps" folders